MAINTAINERS.md

```diff
@@ -79,6 +80,53 @@ The process of becoming a Domain maintainer is:
 | @uxlfoundation/onemath-sparse-write | oneMath Sparse Algebra maintainers |
 | @uxlfoundation/onemath-vm-write | oneMath Vector Math maintainers |
 
+## Backend Maintainer
+
+Backend maintainer has responsibility for a specific backend in the project.
+Backend maintainers are collectively responsible for developing and maintaining their backends,
+including reviewing certain changes to their backend and indicating
+whether those changes are ready to merge. They have a track record of
+contribution and review in the project.
+
+Responsibilities:
+  * Follow the [Code of Conduct](CODE_OF_CONDUCT.md).
+  * Follow and enforce the project [contributing guidelines](CONTRIBUTING.md).
+  * Co-own with other backend maintainers a specific backend, including contributing
+    bug fixes, implementing features, and answering backend specific questions
+    in [#onemath](https://uxlfoundation.slack.com/archives/onemath) Slack channel.
+  * Review pull requests pertaining to their specific backend.
+  * Monitor testing results and flag issues in their specific areas of
+    responsibility.
+  * Support and guide Contributors.
+
+Requirements:
+  * Experience as Contributor in the specific backend for at least 6 months.
+  * Commit at least 25% of working time to the project.
```

Comment:
Should we include this requirement, or just use the one below that says "Commits to being responsible for that specific backend" ?

Reply:
Good point! Removed the 25% commitment as likely there would not be as much work required for backend maintainers.